nixos/doc/manual/from_md/release-notes/rl-2111.section.xml

```diff
@@ -1376,6 +1376,17 @@ Superuser created successfully.
           for those who want to have all RetroArch cores available.
         </para>
       </listitem>
+      <listitem>
+        <para>
+          <literal>yarn2nix-moretea.spdxLicense</literal> is now renamed
+          to <literal>yarn2nix-moretea.getLicenseFromSpdxId</literal>,
+          and is now derived from the new
+          <literal>lib.meta.getLicenseFromSpdxId</literal>. The renamed
+          function is only referenced by
+          <literal>yarn2nix-moretea.mkYarnPackage</literal> in Nixpkgs
+          so far.
```

Comment:
This information will get oudated quickly, and isn't that useful for the release notes IMO.
```suggestion
          <literal>lib.meta.getLicenseFromSpdxId</literal>.
```